tests/python/contrib/test_cmsisnn/test_binary_ops.py

```diff
@@ -46,12 +54,26 @@ def make_model(
     input_1_zero_point,
     out_scale=1.0 / 256,
     out_zero_point=-128,
+    input_0_type=BinaryOpInputType.Variable,
+    input_1_type=BinaryOpInputType.Variable,
 ):
     """Create a Relay Function / network model"""
 
+    def create_input(name, input_type, shape, dtype):
```

Comment:
Does this have to be nested inside `make_model`? We should also consider if we want this many branches in our test functions, you could parameterise with input creation functions to flatten out the logic.

Reply:
As discussed offline, I would make the change to parameterize it.